ngx_http_vod_module.c

```diff
@@ -4547,9 +4547,12 @@ ngx_http_vod_map_media_set_apply(ngx_http_vod_ctx_t *ctx, ngx_str_t* mapping, in
 			sequence->mapped_uri = mapped_source->mapped_uri;
 			sequence->language = mapped_media_set.sequences->language;
 			sequence->label = mapped_media_set.sequences->label;
+                        sequence->id = mapped_media_set.sequences->id;
 			cur_source->mapped_uri = mapped_source->mapped_uri;
 			cur_source->encryption_key = mapped_source->encryption_key;
 
+                        ctx->submodule_context.media_set.id = mapped_media_set.id;
```

Comment:
For the sake of good order, please move this line to before 'sequence = cur_source->sequence'. This way it will be from the top level object to the lower level objects (media set->sequence->source)